app/javascript/packs/admin/users/memberIndex.js

```diff
@@ -3,6 +3,54 @@ import { openDropdown, closeDropdown } from '@utilities/dropdownUtils';
 import { copyToClipboard } from '@utilities/runtime';
 import { showWindowModal, closeWindowModal } from '@utilities/showModal';
 
+/**
+ * Adds a single event listener to the "applied filters" section, allowing the user to remove a given filter.
+ *
+ * NB: We use buttons instead of links, since the redirection to a new URL is more of an implementation detail than something
```

Comment:
What does NB stand for here - non-blocking?

Reply:
TIL this might just be a British thing 😆  https://www.macmillandictionary.com/dictionary/british/nb

Reply:
(I'll change it for clarity!)

Reply:
So interesting! I like it. 😄